vendor/wp-now/src/execute-wp-cli.ts

```diff
@@ -24,20 +22,51 @@ export async function executeWPCli(args: string[]) {
 	});
 	const [, php] = phpInstances;
 
+	//Set the SAPI name to cli before running the script
+	await php.setSapiName('cli');
```

Comment:
This was needed to set the SAPI to CLI as without it, I was getting the error:

```
WP-CLI only works correctly from the command line, using the 'cli' PHP SAPI.
You're currently executing the WP-CLI binary via the 'wasm' PHP SAPI.
In case you were trying to run this file with a web browser, know that this cannot technically work.
```

Kudos to @sejas for finding this approach to set SAPI.